crates/containerd-shim-wasm/src/sandbox/shim.rs

```diff
@@ -1385,6 +1400,57 @@ where
     }
 }
 
+pub fn parse_mount(m: &Mount) -> MsFlags {
```

Comment:
Can we use https://github.com/containers/youki/blob/6d698da2c70ca7bfa71699495157acf3d9930657/crates/libcontainer/src/rootfs/utils.rs#L86?

Reply:
To note, I'm generally ok with copying, but I think we do want to move as much of our system level code to what's provided in libcontainer.

If we can't use it outright, it may be helpful to extract this into a separate module to keep as much as we can together.... just a thought.

Reply:
I move the copy code to local crate `thirdparty` a07e9c3def1356f418e4628eaa07fa367adacbfc